blueoil/converter/core/operators.py

```diff
@@ -814,7 +814,8 @@ def _check_consistency(self) -> None:
             2. (kernel_size^2 * {8, 16}).
         """
         super()._check_consistency()
-        if self.channel % 32 != 0:
+        if self.input_ops['input'].op_type == 'LinearMidTreadHalfQuantizer' and \
```

Comment:
@tk26eng 
Thanks!

If characters less than 120, it's better to avoid `\`.
And please fix the documentation about `space_to_depth`.
https://docs.blueoil.org/converter/supported_ops.html#

Reply:
@iizukak 
Those have been fixed.
Please check them again.